react/tic-tac-toe/src/app/screens/Game/components/Board/index.js

```diff
@@ -1,24 +1,15 @@
 import React, { Component } from 'react';
-import { array, func } from 'prop-types';
 
 import Square from '../Square';
-
+import { number } from 'prop-types';
 import styles from './styles.module.scss';
 
 class Board extends Component {
 
   renderSquare(i) {
-    const {squares, onClick} = this.props;
-    return (
-      <Square
-        value={squares[i]}
-        onClick={onClick}
-        squareNumber={i}
-      />
-    );
+    return <Square id={i} />;
```

Comment:
I think that is not necessary extract this to function. Return directly without doing this.renderSquare(i)